packages/next/src/generators/react-query/generator.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    formatFiles,
+    formatFilesWithEslint,
+    hasGeneratorExecutedForProject,
+    tsMorphTree,
+    verifyPluginCanBeInstalled,
+} from '@ensono-stacks/core';
+import {
+    GeneratorCallback,
+    joinPathFragments,
+    logger,
+    readJson,
+    readProjectConfiguration,
+    Tree,
+} from '@nx/devkit';
+import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
```

Comment:
Believe this is a deprecated function and has been moved to @nx/devkit, can we change the import location?